PORTAL/jobs/__init__.py

```diff
@@ -278,28 +290,30 @@ def ensure_next(self) -> None:
             logger.debug('A job is already running (and will kick off the next one from the queue)')
             # XXX Check the pidfile.
             return
-        queue = self.queue.snapshot
-        if queue.paused:
-            logger.debug('No job is running but the queue is paused')
-            return
-        if not queue:
-            logger.debug('No job is running and none are queued')
-            return
-        # Run in the background.
-        cfgfile = self._cfg.filename
-        if not cfgfile:
-            raise NotImplementedError
-        logger.debug('No job is running so we will run the next one from the queue')
-        _utils.run_bg(
-            [
-                sys.executable, '-u', '-m', 'jobs', '-v',
-                'internal-run-next',
-                '--config', cfgfile,
-                #'--logfile', self._fs.queue.log,
-            ],
-            logfile=self._fs.queue.log,
-            cwd=_common.SYS_PATH_ENTRY,
-        )
+        for queue in self.queues.iter_queues():
+            _queue = queue.snapshot
```

Comment:
FWIW, I personally would have flipped this around (since "queue" is what actually gets used in the loop.

```suggestion
        for _queue in self.queues.iter_queues():
            queue = queue.snapshot
```